app/views/layouts/_nav.html.slim

```diff
@@ -4,8 +4,11 @@
       .navbar-item
         = image_tag "railslink-icon.png", alt: "", id: "logo"
         b= configatron.app_name
+      a.navbar-burger.burger aria-expanded="false" aria-label="menu" data-target="navbarBasicExample" role="button"
```

Comment:
Is the data-target attribute necessary?

Reply:
Yeah, that has to link with navbar-menu class.